api/v1alpha1/flowcollector_types.go

```diff
@@ -219,19 +222,63 @@ type FlowCollectorEBPF struct {
 type FlowCollectorKafka struct {
 	// Important: Run "make generate" to regenerate code after modifying this file
 
+	// InstanceSpec is the Spec section of LokiStack instance.
+	// Experimental feature. If provided, the Loki operator will be automatically installed
+	// using operator lifecycle manager (https://olm.operatorframework.io/)
+	// It will create a "Subscription" using "operators.coreos.com/v1alpha1"
+	// and "Kafka" instance, "KafkaTopic", "KafkaUser" using "kafka.strimzi.io/v1beta2"
+	InstanceSpec *KafkaInstanceSpec `json:"instanceSpec,omitempty"`
```

Comment:
`instanceSpec` seems something too generic and it's difficult to know what refers to.

I'd add something more related with the intention of let the NOO to deploy it, like: `autoDeploy`, `autoCreate`, `fromOperator`....

Reply:
If everyone agree we can move to `autoDeploy` :+1: 

^^^ 
@jotak @OlivierCazade

Reply:
moved to `operatorsAutoInstall` array + `autoInstallSpec` sections